CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 ## Unreleased
 
+- BREAKING: `incident_catalog_type`'s `source_repo_url` attribute is now
+  required. This prevents the catalog type from being edited manually, and ensures
+  there is a link from the incident.io dashboard to the configuration that defines
+  the catalog type.
+- Add `schema_only` as an option on `incident_catalog_type_attribute`. When set
+  to `true`, the attribute will be created from Terraform, but values for it must
+  be edited in the incident.io dashboard.
+- Add `managed_attributes` to both `incident_catalog_entry` and `incident_catalog_entries`. This
+  allows you to manage only some attributes of a catalog entry, while leaving others
+  unchanged. When attributes on a type are marked as `schema_only`, this avoids
```

Comment:
This is a bit unclear to me. Do they _need_ to use schema_only on the attribute to be able to use this?